src/main/java/com/denizenscript/depenizen/bukkit/properties/jobs/JobsJobProperties.java

```diff
@@ -4,34 +4,34 @@
 import com.denizenscript.denizencore.objects.Mechanism;
 import com.denizenscript.denizencore.objects.ObjectTag;
 import com.denizenscript.denizencore.objects.properties.Property;
-import com.denizenscript.denizencore.tags.Attribute;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
 import com.denizenscript.depenizen.bukkit.objects.jobs.JobsJobTag;
 
-public class JobPlayer implements Property {
-
+public class JobsJobProperties implements Property {
     public static boolean describes(ObjectTag job) {
         return job instanceof JobsJobTag;
     }
 
-    public static JobPlayer getFrom(ObjectTag job) {
+    public static JobsJobProperties getFrom(ObjectTag job) {
```

Comment:
This rename doesn't make sense. The original name was correct, the new name is not.